pkg/printers/internalversion/additional_printers_test.go

```diff
@@ -274,157 +258,25 @@ func TestNamePrinter(t *testing.T) {
-	table := map[string]struct {
-		pod    v1.Pod
-		expect string
-	}{
-		"nilInfo":   {v1.Pod{}, "false"},
-		"emptyInfo": {v1.Pod{Status: v1.PodStatus{ContainerStatuses: []v1.ContainerStatus{}}}, "false"},
-		"fooExists": {
-			v1.Pod{
-				Status: v1.PodStatus{
-					ContainerStatuses: []v1.ContainerStatus{
-						{
-							Name: "foo",
-						},
-					},
-				},
-			},
-			"false",
-		},
-		"barExists": {
-			v1.Pod{
-				Status: v1.PodStatus{
-					ContainerStatuses: []v1.ContainerStatus{
-						{
-							Name: "bar",
-						},
-					},
-				},
-			},
-			"false",
-		},
-		"bothExist": {
-			v1.Pod{
-				Status: v1.PodStatus{
-					ContainerStatuses: []v1.ContainerStatus{
-						{
-							Name: "foo",
-						},
-						{
-							Name: "bar",
-						},
-					},
-				},
-			},
-			"false",
-		},
-		"barValid": {
-			v1.Pod{
-				Status: v1.PodStatus{
-					ContainerStatuses: []v1.ContainerStatus{
-						{
-							Name: "foo",
-						},
-						{
-							Name: "bar",
-							State: v1.ContainerState{
-								Running: &v1.ContainerStateRunning{
-									StartedAt: metav1.Time{},
-								},
-							},
-						},
-					},
-				},
-			},
-			"false",
-		},
-		"bothValid": {
-			v1.Pod{
-				Status: v1.PodStatus{
-					ContainerStatuses: []v1.ContainerStatus{
-						{
-							Name: "foo",
-							State: v1.ContainerState{
-								Running: &v1.ContainerStateRunning{
-									StartedAt: metav1.Time{},
-								},
-							},
-						},
-						{
-							Name: "bar",
-							State: v1.ContainerState{
-								Running: &v1.ContainerStateRunning{
-									StartedAt: metav1.Time{},
-								},
-							},
-						},
-					},
-				},
-			},
-			"true",
-		},
-	}
-	// The point of this test is to verify that the below template works.
-	tmpl := `{{if (exists . "status" "containerStatuses")}}{{range .status.containerStatuses}}{{if (and (eq .name "foo") (exists . "state" "running"))}}true{{end}}{{end}}{{end}}`
-	printer, err := genericprinters.NewGoTemplatePrinter([]byte(tmpl))
-	if err != nil {
-		t.Fatalf("tmpl fail: %v", err)
-	}
-
-	for name, item := range table {
-		buffer := &bytes.Buffer{}
-		err = printer.PrintObj(&item.pod, buffer)
-		if err != nil {
-			t.Errorf("%v: unexpected err: %v", name, err)
-			continue
-		}
-		actual := buffer.String()
-		if len(actual) == 0 {
-			actual = "false"
-		}
-		if e := item.expect; e != actual {
-			t.Errorf("%v: expected %v, got %v", name, e, actual)
-		}
-	}
-}
-
 // TODO(seans3): Move this test to cli-runtime/pkg/printers.
 func TestPrinters(t *testing.T) {
 	om := func(name string) metav1.ObjectMeta { return metav1.ObjectMeta{Name: name} }
 
 	var (
-		err              error
-		templatePrinter  printers.ResourcePrinter
-		templatePrinter2 printers.ResourcePrinter
-		jsonpathPrinter  printers.ResourcePrinter
+		err             error
+		jsonpathPrinter printers.ResourcePrinter
 	)
 
-	templatePrinter, err = genericprinters.NewGoTemplatePrinter([]byte("{{.name}}"))
```

Comment:
did this test get moved? I'm not seeing `NewGoTemplatePrinter([]byte("{{.name}}"))`

Reply:
I removed templatePrinter, since it doesn't have any corresponding errors in "expectedErrors". But I see now that it still tests the templatePrinter (by just ensuring it doesn't error). I will add it back.

Reply:
Added back in a new test: `TestTemplateSuccess`.